src/julia.h

```diff
@@ -1578,13 +1582,13 @@ typedef struct _jl_task_t {
     jl_value_t *backtrace;
     jl_value_t *logstate;
     jl_function_t *start;
-    jl_jmp_buf ctx;
-    size_t bufsz;
-    void *stkbuf;
 
-// hidden fields:
-    size_t ssize;
-    size_t started:1;
+// hidden state:
+    jl_ucontext_t ctx; // saved thread state
+    void *stkbuf; // malloc'd memory (either copybuf or stack)
+    size_t bufsz; // actual sizeof stkbuf
+    unsigned int copy_stack:31; // sizeof stack for copybuf
```

Comment:
Could this be called something like `copy_amount`? This name sounds really similar to the boolean COPY_STACKS.

Reply:
Yeah, it probably could. I use this as a boolean often enough though too (as the runtime version of COPY_STACKS) that I'm being intentionally a bit ambiguous.